### ISSUE TITLE
[Improvement] More user-friendly defaults with tea entry

### ISSUE THREAD

**Issue by JaromirDvoracek** [Sesu8642/InfusionTimer#15]
Hello @Sesu8642,

I have 2 frustrating issues with a tea entry screen (after you tap on the plus in the right bottom corner of the main screen):
1. When I tap outside the entry window, it closes without warning (100 % of mistap in my case)
2. Cancel is on the right, save on the left (I automatically tap on the right leading again to close without save/warning)

Does it make sense to you to change this behavior? 

Cheers

Jaromir

**Comment by Sesu8642** [comment]
Thanks for your feedback!

1. That's default behavior of the dialog. I agree that it's not very useful but can be frustrating if closed accidentally. I changed it.
2. When I looked up images of similar dialogs, there doesn't seem to be a standard way to order the buttons. But it seems that while the cancel option is usually on the right on Windows, on Android, it is on the left more often than not. I switched the button order and made the Save button bold. I think an extra warning before closing might be annoying if you actually want to cancel. Let me know if accidental closing of the dialog still happens to you after the changes.
